CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# dbt_jira_source v0.5.0
+
```

Comment:
## 🎉 Documentation and Feature Updates
- Updated README documentation updates for easier navigation and setup of the dbt package
- Added `jira_[source_table_name]_identifier` variables to allow for easier flexibility of the package to refer to source tables with different names.